model/src/form/form-submission/index.ts

```diff
@@ -0,0 +1,42 @@
+import Joi from 'joi'
+
+import {
+  type SubmitPayload,
+  type SubmitRecord,
+  type SubmitRecordset
+} from '~/src/form/form-submission/types.js'
+
+/**
+ * Joi schema for `SubmitRecord` interface
+ * @see {@link SubmitRecord}
+ */
+export const formSubmitRecordSchema = Joi.object<SubmitRecord>({
+  name: Joi.string().required(),
+  title: Joi.string().required(),
+  value: Joi.string().required()
+})
+
+/**
+ * Joi schema for `SubmitRecordset` interface
+ * @see {@link SubmitRecordset}
+ */
+export const formSubmitRecordsetSchema = Joi.object<SubmitRecordset>({
+  name: Joi.string().required(),
+  title: Joi.string().required(),
+  value: Joi.array()
```

Comment:
Do we need to type these arrays like elsewhere?

```suggestion
  value: Joi.array<XXX>()
```

Reply:
Resolved with 9e45b534c7586486c57786c30cc827ac19081d69